imod_coupler/drivers/dfm_metamod/mapping.py

```diff
@@ -166,10 +166,11 @@ def mapping_active_mf_dflow1d(
             ndmin=2,
             skiprows=1,
         )
+        ptx = table_active_mfriv2dflow1d[:, 0]
+        pty = table_active_mfriv2dflow1d[:, 1]
         mf_idx = table_active_mfriv2dflow1d[:, 2].astype(int) - 1
-        dflow_idx = np.array(
-            [self.dflow1d_lookup[row[0], row[1]] for row in table_active_mfriv2dflow1d]
-        )
+        dist, dflow_idx = self.dflow1d_lookup.query(np.c_[ptx, pty])
```

Comment:
if the " dist" variables are not used, you can also write an underscore, as in 
_,dflow_idx = self.dflow1d_lookup.query(np.c_[ptx, pty])